plugins/command-manager/src/main/java/com/wazuh/commandmanager/rest/RestPostCommandAction.java

```diff
@@ -94,42 +95,67 @@ private RestChannelConsumer handlePost(RestRequest request) throws IOException {
                 request.uri(),
                 request.getRequestId(),
                 request.header("Host"));
+
         // Get request details
+        if (!request.hasContent()) {
+            // Manejar el caso cuando no hay contenido
+            log.error("Body content is required");
+            return null;
```

Comment:
Instead of returning `null`, it would be better to reply to the request with an error status.

https://github.com/wazuh/wazuh-indexer-plugins/blob/07a1870d44ac6c9f0344e027498e5b133dd26e53/plugins/command-manager/src/main/java/com/wazuh/commandmanager/rest/RestPostCommandAction.java#L171-L177